### ISSUE TITLE
Improve user experience with command line params

### ISSUE THREAD

**Issue by larsen0815** [OpenVPN/openvpn-gui#172]
Regarding the param "--silent_connection" to openvpn-gui.exe, the help says:
````
--silent_connection		: 1=Do not show the status dialog while connecting or non-critical warnings at startup.
````

I guess that most users like me think that you have to use "--silent_connection", when actually you have to use "--silent_connection 1". Took me a while to figure this out, especially as the error message was misleading (didn't think of a parameter missing to a parameter).
````
Options error: Unrecognized option or missing parameter(s): --silent_connection
Use openvpn-gui --help for more info.
````

IMHO "--silent_connection" (without explicitly using "1") should be sufficient and use "1" as its default value. This style is used in every other tool I know, so for a better user experience, it should additionally be possible to leave out the following "1". Same goes for other comparable params.

**Comment by otaruMendez** [comment]
I think this is not really an issue as it is explicitly stated in the help window.

**Comment by larsen0815** [comment]
I explicitly explained why it is an issue although it is explained in the help window...

**Comment by mattock** [comment]
I agree with @larsen0815. I would expect a more "standard" (discussion [here](https://stackoverflow.com/questions/9725675/is-there-a-standard-format-for-command-line-shell-help-text)) way to define the parameters. For example,  "openvpn --help" handles such cases like this
```
--plugin m [str]: Load plug-in module m passing str as an argument
                  to its initialization function.
```
So "m" is a mandatory parameter and "[str]" is optional.

In this simplest fix would be
```
--silent_connection arg : 1=Do not show the status dialog while connecting or non-critical warnings at startup.
```
It's not perfect, but at least gives a clear clue that an extra arg is required. If the code is not incredibly funky then implementing this should be easy even for a non-programmer.

That said, I think that these on/off (boolean) parameters should not need any arguments. Rather we should have sane defaults. In this case default to non-silent connection and allow enabling silent connection with "--silent_connection" (without the "1").
